cmd/broker/ingress/main.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/google/knative-gcp/pkg/broker/ingress"
+	"go.uber.org/zap"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/signals"
+)
+
+func main() {
+	// Since we pass nil, a default config with no error will be returned.
+	cfg, _ := logging.NewConfigFromMap(nil)
+	logger, _ := logging.NewLoggerFromConfig(cfg, "broker-ingress")
+	ctx := signals.NewContext()
+	logging.WithLogger(ctx, logger)
+
+	ingress, err := ingress.NewHandler(ctx)
+	if err != nil {
+		logger.Fatal("Unable to create ingress handler: ", zap.Error(err))
```

Comment:
```suggestion
		logger.Fatalw("Unable to create ingress handler: ", zap.Error(err))
```

The alternative to these is using `knative.dev/eventing/pkg/logging`, instead of `knative.dev/pkg/logging`, but that would require changing lines 28-29.

Or adding `Desugar()` to each call.

Reply:
I prefer the Desugared logger interface myself. You could just add this line after calling logging.WithLogger:

```go
logger = logger.Desugar()
```

Reply:
This is in the setup path and performance is not critical, so I guess it's OK to use the sugared logger? I am also OK to change it to desugar though.

With regards to `Fatalw`, does that require key-value pairs? I am new to zap and not sure how does it look like.

Reply:
Changed to `Desugar`.